app/models/github.js

```diff
@@ -0,0 +1,34 @@
+const config = require('../config');
+const { Model, ModelSet } = require('./base');
+const github = require('../api_clients/github');
+
+
+class Repo extends Model {
+  static list(params = {}) {
+    return Promise.all(
+      config.github.orgs.map((org) => {
+        return github.api.repos.getForOrg({
+          org,
+          type: 'all',
+          page: params.page || 1,
+          per_page: params.per_page || 500
+        });
+      }))
+      .then((results) => {
+        return new ModelSet(
+          this.prototype.constructor,
+          [].concat(...results.map(result => result.data))
```

Comment:
I may miss something here so apologies if it's silly.

Isn't this `[].concat()` equivalent to `results.map(result => result.data)` (which should map to an array, unless results is `null` or something like that).

Reply:
`results` is an array of objects with a `data` array property, eg: `[{ data: [1, 2] }, { data: [3, 4] }]`.

I do `results.map(result => result.data)`, which gives me an array of arrays, eg: `[ [1, 2], [3, 4] ]`.

Then I do `[].concat(...<array of arrays>)`, which concats them all into one array, eg: `[1, 2, 3, 4]`.

Reply:
👍